spin/cmds/meson.py

```diff
@@ -215,8 +224,13 @@ def _get_configured_command(command_name):
     is_flag=True,
     help="Generate a coverage report of executed tests. An HTML copy of the report is written to `build/coverage`.",
 )
+@click.option(
+    "--gcov",
+    is_flag=True,
+    help="Enable C code coverage via gcov (requires GCC). gcov output goes to build/**/*.gc*",
```

Comment:
Should we advise, somewhere, on how to display those reports?

Reply:
Added via [62891ad](https://github.com/scientific-python/spin/pull/146/commits/62891ad92f394782c19070a13dab472d83c3274a)